docs/docker.md

```diff
@@ -0,0 +1,210 @@
+# Contenedor de pruebas
+
+En anteriores commits había información sobre Docker y algunas de sus alternativas pero como el objetivo indica que se use Docker se ha eliminado este apartado y evitar relleno.
+
+## Integración de Docker en nuestro proyecto
+
+### Cuestiones a tener en cuenta en la elección de una imagen
+
+Nos vamos a encontrar, en general, con dos tipos de imagen distintas, una basada en debian y otra en Alpine. Las diferencias entre estas dos son las siguientes:
+
+* El tamaño entre una y otra. Alpine tiene un tamaño menor que Debian. Con esto conseguimos que operaciones como construir, pull y push sean más rápidas.
+* Consumir menos memoria por el propio sistema operativo en comparación con Debian. Alpine se considera seguro y rápido.
```

Comment:
¿Y Debian es lento e inseguro? ¿Podéis intentar establecer unos criterios previos a la elección?